### ISSUE TITLE
npm package tarball still contains wrong "dependencies" of mongoose and others.

### ISSUE THREAD

**Issue by yoadsn** [nodkz/graphql-compose-mongoose#14]
Don't know how this weird thing happened since I can see the updated package.json in Github does not have "mongoose" in the "dependencies" section.
But when doing "npm view graphql-compose-mongoose" it is clear that the package.json in the registry still has this dependencies section:

```
 dependencies:
   { 'babel-runtime': '^6.20.0',
     'graphql-compose': '^1.7.0',
     'graphql-compose-connection': '^2.1.2',
     mongoose: '^4.7.4',
     'object-path': '^0.11.3' },
```

Not sure how critical this is but it caused some trouble for me - same is with `graphql-compose` which has `graphql` in it's dependencies section.
Probably has to do with how the package was updated in npm once those dependencies were converted to peer dependencies.

**Comment by nodkz** [comment]
Aggr, very strange. May be npm has a bug when the package was published. 
v1.4.1 does not have such problem.

**Comment by nodkz** [comment]
Does not helps
in v1.4.3 problem remained

```
  dependencies:
   { 'babel-runtime': '^6.20.0',
     'graphql-compose': '^1.8.0',
     'graphql-compose-connection': '^2.1.2',
     mongoose: '^4.7.4',
     'object-path': '^0.11.3' },
  peerDependencies:
   { graphql: '>=0.7.1 || >=0.8.0',
     'graphql-compose': '>=1.4.0',
     'graphql-compose-connection': '>=2.0.2',
     mongoose: '>=4.0.0' },
```

**Comment by nodkz** [comment]
Try to find time in the coming days and solve this problem. If you'll found some talks, issues about this problem with npmjs, please share links.

Right now you may use 1.4.1 or clone package in run `npm build && npm pack` put this file to your app and write path to this file in package.json

**Comment by yoadsn** [comment]
Will look into it some more.
For anyone who encounters problems with wrong expected types which look like invariant errors from graphql.js or warning from graphql-compose - You might have a local installation of either:
`graphql-compose` inside the installation of `graphql-coompose-mongoose`
`graphql` inside the installation of `graphql-compose`.
To prevent this from happening - make sure you delete the module folders, and install the peer dependencies **in the correct expected version** otherwise npm would install the local package again.

**Comment by nodkz** [comment]
Fixed in `1.4.5` via https://github.com/nodkz/graphql-compose-mongoose/commit/eaa5fc67ea9bf2077a8cb04645d49ea79fb9b75e

Also upgrade `graphql-compose` till 1.8.1
